src/System.Private.CoreLib/shared/System/Globalization/CultureData.cs

```diff
@@ -812,27 +802,15 @@ internal string CultureName
             }
         }
 
-        // Are overrides enabled?
-        internal bool UseUserOverride
-        {
-            get
-            {
-                return _bUseOverrides;
-            }
-        }
+        /// <summary>
+        /// Are overrides enabled?
+        /// </summary>
+        internal bool UseUserOverride => _bUseOverrides;
 
-        // locale name (ie: de-DE, NO sort information)
-        internal string SNAME
-        {
-            get
-            {
-                if (_sName == null)
-                {
-                    _sName = string.Empty;
-                }
-                return _sName;
-            }
-        }
+        /// <summary>
+        /// locale name (ie: de-DE, NO sort information)
+        /// </summary>
+        internal string Name => _sName ?? string.Empty;
 
         // Parent name (which may be a custom locale/culture)
         internal string SPARENT
```

Comment:
>SPARENT [](start = 24, length = 7)

why did you change some and left the rest? I am not sure we need to have a mix from the modern and old names here.

Reply:
Fixed all